internal/cli/plugin/first_class.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+
+	"github.com/google/go-github/v61/github"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/plugin"
+	"github.com/spf13/cobra"
+)
+
+const (
+	SourceType = "first-class"
+)
+
+//go:embed first-class-plugins.json
+var firstClassPluginsFile []byte
```

Comment:
[q] is there any reason to use this approach over having a static variable in the code?

This approach can fail at runtime when the JSON contains errors.

Reply:
Changed it to use a static variable instead